.remarkrc

```diff
@@ -0,0 +1,15 @@
+{
+  "plugins": {
+    "remark-lint": {
+      "fenced-code-flag": {
+        "allowEmpty": true
+      },
+      "list-item-indent": "space",
+      "maximum-heading-length": 999,
+      "maximum-line-length": 9999,
```

Comment:
You should be able to pass `false` to ignore this rule, which looks neater than specifying a large number :)